.gitignore

```diff
@@ -3,3 +3,4 @@
 terraform.tfstate*
 *.csv
 .DS_Store
+streamlit_template/.streamlit/secrets.toml
```

Comment:
Have `git` ignore any changes to `secrets.toml` to avoid developers accidentally committing their Snowflake credentials to the repository. Instead we use `example_secrets.toml` that people can copy from to create their own setup.

Reply:
Nit: I would add py_cache to the .gitignore